### PR TITLE
Audit dovecot/centos article: INFODEV-1595

### PR DESCRIPTION
**Pull request by RackerKate**

### CODE REVIEW

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-8282269055/how-to/). :bow:

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-3e7b416c51/how-to/).

**Comment by cyrichardson** [comment]
On it, @RackerKate.

**Comment by cyrichardson** [comment]
@RackerKate Just a few suggestions here: 
[Install-and-configure-Dovecot-on-CentOS_cyrEdits.docx](https://github.com/rackerlabs/rackspace-how-to/files/2493361/Install-and-configure-Dovecot-on-CentOS_cyrEdits.docx)

Great job on the rewrite!

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-8bf6e3ea64/how-to/)

File: content/cloud-servers/dovecot-installation-and-configuration-on-centos.md [hunk 1]
```diff
@@ -1,38 +1,36 @@
...
 
--   Operating System and version: Linux - CentOS 6.0 and above.
--   Postfix installed.
--   Intended purpose for document: Basic installation and configuration
-    of Dovecot.
--   Target Audience: System Administrators, Mail
-    Administrators, Postmasters.
+You need the following operating system and software to use Dovecot:
+
+- A CentOS&reg; 6.0 or above Linux distribution
```
Comment (catlook) [inline content/cloud-servers/dovecot-installation-and-configuration-on-centos.md]: Per style guide: Use earlier, later | To refer to product releases (version numbers).Note: Don't use higher, lower, above, below, older, or newer.

File: content/cloud-servers/dovecot-installation-and-configuration-on-centos.md [hunk 2]
```diff
@@ -1,38 +1,36 @@
...
+last_modified_date: '2018-10-18'
+last_modified_by: Kate Dougherty
 product: Cloud Servers
 product_url: cloud-servers
 ---
 
-If you've installed Postfix to operate as the SMTP service on a would-be
-email server you might still need a way to retrieve the incoming mail
-from your server.
+If you have installed the [Postfix](http://www.postfix.org) mail server to
```
Comment (catlook) [inline content/cloud-servers/dovecot-installation-and-configuration-on-centos.md]: trademark or reg mark Dovecot and Postfix if necessary!

**Pull request merged**